plugin.py

```diff
@@ -31,7 +31,11 @@ def plugin_info(self):
         return plugin_info
 
     def process(self, trace, data_context):
-        kaitai_utils.write_kaitai_to_trace(trace)
+        # byte arrays shorter than byte_array_length are written as hex in the resulting JSON, longer ones are written
+        # to child traces. Change it if necessary!
+        byte_array_length = 256
```

Comment:
Like @remco-nfi commented [here](https://github.com/NetherlandsForensicInstitute/hansken-extraction-plugin-template-kaitai/pull/7#discussion_r1743859630), it might be nice to make this a constant at the top of the class, so it's easier to find.

Reply:
Done as discussed!